### PR TITLE
[INLONG-2869][Manager] Support config sync send data for agent and sort

### LINKED ISSUE #2869: [Feature][Manager] Support config sync send data for agent and sort
### Description

_No response_

### Use case

_No response_

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by binaryWorld**
### Title Name: [INLONG-2869][Manager] Support config sync send data for agent and sort

Fixes #2869 

### Motivation

Support config sync send data for agent and sort by adding `syncSend` field for inlong_stream.

### Modifications

*Describe the modifications you've done.*

### Verifying this change

- [X] Make sure that the change passes the CI checks.

### Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: inlong-manager/manager-web/sql/apache_inlong_manager.sql [hunk 1]
```diff
@@ -373,6 +373,7 @@ CREATE TABLE `inlong_stream`
     `data_separator`         varchar(8)        DEFAULT NULL COMMENT 'The source data field separator, stored as ASCII code',
     `data_escape_char`       varchar(8)        DEFAULT NULL COMMENT 'Source data field escape character, the default is NULL (NULL), stored as 1 character',
     `have_predefined_fields` tinyint(1)        DEFAULT '0' COMMENT '(File, DB access) whether there are predefined fields, 0: none, 1: yes (save to inlong_stream_field)',
+    `sync_send`              tinyint(1)        DEFAULT '0' COMMENT 'order_preserving 0: none, 1: yes',
```
Comment (healchow) [inline inlong-manager/manager-web/sql/apache_inlong_manager.sql:376]: Please add this field in another SQL file in the manager-test module.

Comment (binaryWorld) [reply inlong-manager/manager-web/sql/apache_inlong_manager.sql:376]: done

File: inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java [hunk 2]
```diff
@@ -112,24 +117,27 @@ private TaskResult getTaskResult(TaskRequest request) {
...
+            dataConfig.setInlongStreamId(inlongStreamId);
             dataConfig.setIp(entity.getAgentIp());
             dataConfig.setUuid(entity.getUuid());
             dataConfig.setExtParams(entity.getExtParams());
             dataConfig.setSnapshot(entity.getSnapshot());
-            return dataConfig;
-        }).collect(Collectors.toList());
-
+            InlongStreamEntity inlongStreamEntity = inlongStreamMapper.selectByIdentifier(inlongGroupId,inlongStreamId);
+            inlongStreamEntity.setSyncSend(inlongStreamEntity.getSyncSend());
```
Comment (healchow) [inline inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AgentServiceImpl.java:138]: It should be set in dataConfig, not stream entity.

**Review by healchow — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]
LGTM

**Pull request merged**